src/gui/CMakeLists.txt

```diff
@@ -234,14 +234,13 @@ if(UNIX AND NOT APPLE)
 endif()
 
 if(NOT BUILD_OWNCLOUD_OSX_BUNDLE)
-
     if(NOT WIN32)
-        file( GLOB _icons "${theme_dir}/colored/${APPLICATION_ICON_NAME}-icon-*.png" )
-        foreach( _file ${_icons} )
-            string( REPLACE "${theme_dir}/colored/${APPLICATION_ICON_NAME}-icon-" "" _res ${_file} )
-            string( REPLACE ".png" "" _res ${_res} )
-        install( FILES ${_file} RENAME ${APPLICATION_ICON_NAME}.png DESTINATION ${DATADIR}/icons/hicolor/${_res}x${_res}/apps )
-        endforeach( _file )
+        file(GLOB _icons "${theme_dir}/colored/*-${APPLICATION_ICON_NAME}-icon.png")
+        foreach(_file ${_icons})
+            string(REPLACE "${theme_dir}/colored/" "" _res ${_file})
+            string(REPLACE "-${APPLICATION_ICON_NAME}-icon.png" "" _res ${_res})
```

Comment:
These two lines could be done in one line using a regexp

Reply:
Yup, i just ported the existing code tho :)

Reply:
Patches welcome ;-)